templates/basic-ts/tsconfig.json

```diff
@@ -1,25 +1,74 @@
 {
   "compilerOptions": {
-    "allowJs": false,
-    "lib": ["es2015", "es2017"],
-    "module": "commonjs",
-    "moduleResolution": "node",
-    "target": "es5",
-    "noImplicitReturns": true,
-    "noFallthroughCasesInSwitch": true,
-    "noUnusedLocals": false,
-    "pretty": true,
-    "strict": true,
-    "sourceMap": true,
-    "outDir": "./lib",
-    "skipLibCheck": true,
-    "noImplicitAny": true,
-    "esModuleInterop": true,
-    "declaration": true,
-    "resolveJsonModule": true
+    /* Basic Options */
```

Comment:
This looks awesome. The config options with descriptions is a really nice idea.

Reply:
this is what `tsc --init` produce ☺️